### DISCUSSION TITLE
Possibility for both way controller? instead of one way Server-Client controller?

### DISCUSSION THREAD

**Discussion by ibnummuhammad** [input-leap/input-leap#1955] [Feature Requests]
Hi, I'm a huge fan of this KVM switch controller since Barrier, and have been using this heavily on my daily work setup.

Just wondering, is it possible for this KVM switch to do both way from 2 computers? like, you can control computer B's mouse and keyboard from computer A, and you can control computer A's mouse and keyboard from computer B as well. Instead of Server-Client style that only can control from one-way?

**Comment by shymega** [comment]
No, not possible. I did briefly work on a hybrid KVM, but didn't get very far.

**Comment by DareDevil2000** [comment]
I also thought about this , Similar to windows MWB software

**Comment by CJSen** [comment]
I known a app，sharemouse，it can implement this feature, but it is paid, you can try it for free.
